examples/ip_adapter/image_to_image_sd.py

```diff
@@ -144,7 +160,12 @@ def main(args):
 
     # set ms context
     device_id = int(os.getenv("DEVICE_ID", 0))
-    ms.set_context(mode=args.ms_mode, device_target="Ascend", device_id=device_id)
+    ms.set_context(
+        mode=args.ms_mode,
+        device_target="Ascend",
+        device_id=device_id,
+        ascend_config=dict(precision_mode="must_keep_origin_dtype"),
```

Comment:
Why is the precision mode "must_keep_origin_dtype"？ Does the default "force_fp16" cause any artifacts during your experiments?

Reply:
I have encountered the problem of fp16 overflow on SDXL when using full fp32 mode, which result in color shift in the generated images. For the safetly, I set all training/evaluation scripts in this project to use `"must_keep_origin_dtype" to make sure the network type is what we expected.

Reply:
Does it mean if we don't set `precision_mode`, the generated images will be degraded?

Reply:
For IPAdpater + SDXL, the result will be degraded without setting the mode.